doc: fix documentation for `--run` (#53976)
nodejs/node: doc/api/cli.md

> Stability: 1.2 - Release candidate

This runs a specified command from a package.json's `"scripts"` object.

> If an invalid `"command"` is provided, it will list the available scripts.

Reviewer: ```suggestion
If a missing `"command"` is provided, it will list the available scripts.
```

Reviewer 2: I think "invalid" is better. `node --run` is still missing a command so it is not different from the original wording.

Reviewer 2: How about "If there is not a matching "command", it will list the available scripts."?
